contracts/swappers/Leverage/StkFrax3CrvLevSwapper.sol

```diff
@@ -0,0 +1,581 @@
+// SPDX-License-Identifier: MIXED
+
+// File @boringcrypto/boring-solidity/contracts/libraries/BoringMath.sol@v1.2.2
```

Comment:
I would prefer if you would use unflattened files so the main function is easier to read. The main reason we used flattened files in the beginning was for easier manual deployment after having problems with the hardhat etherscan plugin

Reply:
That's a unintentional mistake, it shouldn't be like that I totally agree, I'm fixing it